server/http/helloapp/locations.go

```diff
@@ -37,7 +36,7 @@ func (h *helloapp) HandleLocations(w http.ResponseWriter, req *http.Request, use
 		&apps.ChannelHeaderIconLocation{
 			Location: apps.Location{
 				LocationType: apps.LocationChannelHeaderIcon,
-				Wish:         store.NewWish(AppID, h.AppURL(PathWishPing)),
+				FormURL:      h.AppURL(PathPing),
```

Comment:
Since wishes no longer hold the AppID, we have to think where to include the AppID for the locations, in order for the client to know how to fill up the context of the call. (i.e. the appID)

Reply:
Will add `AppID` elsewhere, tbd.